WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewController.swift

```diff
@@ -1,9 +1,91 @@
 import UIKit
+import Combine
+
+final class CardReaderSettingsViewController: UIViewController {
+    private var subscriptions = Set<AnyCancellable>()
+    private var viewmodel = CardReaderSettingsViewModel()
+    private var subView: UIView?
+    private var alert: UIAlertController?
 
-class CardReaderSettingsViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         configureNavigation()
+
+        self.viewmodel = CardReaderSettingsViewModel()
```

Comment:
it might be interesting to explore injecting this view model trough the constructor.